pkg/controller/azure/compute/compute_suite_test.go

```diff
@@ -168,3 +168,27 @@ func testInstance(p *azurev1alpha1.Provider) *computev1alpha1.AKSCluster {
 		},
 	}
 }
+
+func testInstanceInSubnet(p *azurev1alpha1.Provider) *computev1alpha1.AKSCluster {
+	return &computev1alpha1.AKSCluster{
```

Comment:
instead of duplicating the properties/values from `testInstance`, should this function first call `testInstance()` and then set the `VnetSubnetID` before returning it?

Reply:
Definitely, that would be much more concise. I would like to change these tests in the future as part of #756 to use the `With___()` pattern that is employed on many of the table driven tests ([example](https://github.com/crossplaneio/crossplane/blob/5f4811a895837ceb1240837543414ffda509809b/pkg/controller/aws/cache/managed_test.go#L131)).